<?xml version="1.0" ?>
<launch>

    <param name="filepath" value="$(find osm_planner)/osm_example/botanicka.osm"/>
    <rosparam file="$(find osm_planner)/test/test.yaml" command="load"/>

    <node pkg="osm_planner" name="test_osm" type="test_osm" output="screen"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find osm_planner)/rviz/test.rviz" required="true"/>


</launch>
